web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -142,17 +157,33 @@ const TeamReviews = ({ onBack, periodId }) => {
   const reviewAssignmentsUrl = '/services/review-assignments';
 
   useEffect(() => {
-    loadTeamMembers();
+    loadAssignments();
   }, [currentMembers]);
 
+  useEffect(() => {
```

Comment:
This sets `approvalMode` which determines lots of functionality on this page.
Most of the functionality is the same regardless of the value of this state, so it makes sense to handle both cases in this one component rather than creating two page components.